src/widgetastic_patternfly/__init__.py

```diff
@@ -514,6 +514,7 @@ class BootstrapNav(Widget):
     PARTIAL_TEXT = './/li/a[contains(normalize-space(.), {txt})]'
     ATTR_MATCHING = './/li/a[@{attr}={txt}]'
     TEXT_DISABLED = './/li[contains(@class, "disabled")]/a[text()={txt}]'
+    PARTIAL_TEXT_DISABLED = './/li[contains(@class, "disabled")]/a[contains(text(), {txt})]'
```

Comment:
I don't think it will make any difference, but for consistency with `PARTIAL_TEXT` can we do? 

```suggestion
    PARTIAL_TEXT_DISABLED = './/li[contains(@class, "disabled")]/a[contains(normalize-space(.), {txt})]'
```

Reply:
Good point @john-dupuy, this consistency in normalizing the space should be kept.

Reply:
I think I tried with `normalize-space(.)` at first, but it didn't work as expected. Let me check again though.

Reply:
So I tested again, and it seems to be working. Pushed the changes.